app/src/main/java/com/framgia/music5/screen/playmusic/PlayMusicActivity.java

```
@@ -0,0 +1,210 @@
+            case R.id.image_repeat:
+
+                break;
+        }
+    }
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        return super.onCreateOptionsMenu(menu);
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
+            case android.R.id.home:
+                finish();
+                return true;
+        }
+        return super.onOptionsItemSelected(item);
+    }
+
+    private void initComponents() {
+        mTextCurrentDuration = findViewById(R.id.text_current_duration);
+        mTextDuration = findViewById(R.id.text_duration);
+        mTextTitleSong = findViewById(R.id.text_title_playing);
+        mSeekBar = findViewById(R.id.seek_bar);
+        mButtonShuffle = findViewById(R.id.image_shuffle);
+        mButtonPrevious = findViewById(R.id.image_previous);
+        mButtonState = findViewById(R.id.image_state);
+        mButtonNext = findViewById(R.id.image_next);
+        mButtonRepeat = findViewById(R.id.image_repeat);
+        mHandler = new Handler();
+        Intent intent = new Intent(this, MediaService.class);
+        bindService(intent, mediaConnection, BIND_AUTO_CREATE);
+    }
+
+    private void initListeners() {
+        mButtonShuffle.setOnClickListener(this);
+        mButtonPrevious.setOnClickListener(this);
+        mButtonState.setOnClickListener(this);
+        mButtonNext.setOnClickListener(this);
+        mButtonRepeat.setOnClickListener(this);
+        mSeekBar.setOnSeekBarChangeListener(onSeekChange);
+    }
+
+    private void initToolbar() {
+        Toolbar toolbar = findViewById(R.id.toolbar_play_music);
+        setSupportActionBar(toolbar);
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+        setTitle(getResources().getString(R.string.title_toolbar_playmusic));
+    }
+
+    private String convertToTime(long duration) {
+        SimpleDateFormat sdf = new SimpleDateFormat("mm:ss");
```

WARNING: [SimpleDateFormat] To get local formatting use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()`, or use `new SimpleDateFormat(String template, Locale locale)` with for example `Locale.US` for ASCII dates.
 Almost all callers should use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()` to get a ready-made instance of SimpleDateFormat suitable for the user's locale. The main reason you'd create an instance this class directly is because you need to format/parse a specific machine-readable format, in which case you almost certainly want to explicitly ask for US to ensure that you get ASCII digits (rather than, say, Arabic digits).

Therefore, you should either use the form of the SimpleDateFormat constructor where you pass in an explicit locale, such as Locale.US, or use one of the get instance methods, or suppress this error if really know what you are doing.